datanode/gateway/graphql/schema.graphql

```diff
@@ -656,17 +706,19 @@ type Query {
     pagination: Pagination
   ): FundingPeriodDataPointConnection!
 
-  """
-  Funding payment for perpetual markets.
-  """
-  fundingPayments(
-    "ID of the party"
-    partyId: ID!
-    "ID of the perpetual market to get funding periods for"
-    marketId: ID
+  "Get a list of games and their metrics."
+  games(
+    "Optional game ID to filter for"
+    gameId: ID
+    "Start epoch to return the results from (inclusive), if not provided the most recent epoch will be used."
+    epochFrom: Int
+    "End epoch to return the results to (inclusive), if not provided the most recent epoch will be used."
+    epochTo: Int
+    "Entity scope to filter for, i.e. individual or team. If not provided games for both individuals and teams will be returned."
+    entityScope: EntityScope
     "Pagination control"
     pagination: Pagination
-  ): FundingPaymentConnection!
+  ): GamesConnection!
```

Comment:
Is this change correct?

Reply:
I will double check, it'll be related to my reordering so that the API's are listed alphabetically

Reply:
Confirmed this is correct and due to me moving query definitions around so they are listed alphabetically.